app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,57 @@
+class OrdersController < ApplicationController
+    before_action :order_page_access, only: [:index]
+
+  def index
+    @merchant = session[:user_id]
+    @orders = Order.includes(:order_items).where(order_items: { user_id: @merchant } )
+    # @order_item = OrderItem.find(params[:id])
+
+    if params[:status] == "paid"
```

Comment:
Since the string in this if statement directly correlates to the string in the where query, can you just remove the `if` statement completely and do something like:
```@orders = @orders.where("status = ?", params[:status])```